201-front-door-api-management/azuredeploy.json

```diff
@@ -0,0 +1,1048 @@
+      "metadata": {
+        "description": "Location for the resources."
+      }
+    },
+    "virtualNetwork": {
+      "type": "object",
+      "defaultValue": {
+        "name": "FrontDoorApimSampleVnet",
+        "addressPrefixes": [
+          "10.0.0.0/16"
+        ],
+        "subnets": [
+          {
+            "name": "ApiManagementSubnet",
+            "addressPrefix": "10.0.0.0/24",
+            "nsg": "FrontDoorApimSampleNsg"
+          }
+        ]
+      },
+      "metadata": {
+        "description": "Virtual Network configuration."
+      }
+    },
+    "workspaceName": {
+      "type": "string",
+      "defaultValue": "FrontDoorApimSampleLogAnalytics",
+      "metadata": {
+        "description": "Name of the Log Analytics workspace"
+      }
+    },
+    "workspaceSku": {
+      "type": "string",
+      "allowedValues": [
+        "PerGB2018",
+        "Free",
+        "Standalone",
+        "PerNode",
+        "Standard",
+        "Premium"
+      ],
+      "defaultValue": "PerGB2018",
+      "metadata": {
+        "description": "Pricing tier: PerGB2018 or legacy tiers (Free, Standalone, PerNode, Standard or Premium) which are not available to all customers."
+      }
+    },
+    "appInsightsName": {
+      "type": "string",
+      "defaultValue": "FrontDoorApimSampleAppInsights",
+      "metadata": {
+        "description": "The name of your Application Insights resource used by API Management."
+      }
+    },
+    "apimServiceName": {
+      "type": "string",
+      "defaultValue": "FrontDoorApimSampleApim",
+      "metadata": {
+        "description": "The name of your API Management service."
+      }
+    },
+    "apimPublisherEmail": {
+      "type": "string",
+      "defaultValue": "admin@contoso.com",
+      "metadata": {
+        "description": "The email address of the owner of the service"
+      }
+    },
+    "apimPublisherName": {
+      "type": "string",
+      "defaultValue": "Contoso",
+      "metadata": {
+        "description": "The name of the publisher."
+      }
+    },
+    "apimSku": {
+      "type": "string",
+      "allowedValues": [
+        "Developer",
+        "Premium"
+      ],
+      "defaultValue": "Developer",
+      "metadata": {
+        "description": "The pricing tier of this API Management service. Only Developer and Premium are supported when deploying into a shared VNET."
+      }
+    },
+    "apimCapacity": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Number of deployed units of the SKU."
+      }
+    },
+    "frontDoorName": {
+      "type": "string",
+      "defaultValue": "FrontDoorApimSampleAfd",
+      "minLength": 5,
+      "maxLength": 64,
+      "metadata": {
+        "description": "Name of the Azure Front Door which is globally unique. Min length: 5. Max length: 64"
+      }
+    },
+    "frontDoorEnforceCertificateNameCheck": {
+      "type": "string",
+      "defaultValue": "Disabled",
```

Comment:
Can we enable this by default and/or provide instructions on how to do so?

Reply:
Done.